br/pkg/task/restore.go

```diff
@@ -1108,8 +1108,9 @@ func runSnapshotRestore(c context.Context, mgr *conn.Mgr, g glue.Glue, cmdName s
 	errCh := make(chan error, 32)
 	postHandleCh := afterTableRestoredCh(ctx, createdTables)
 
-	// pipeline checksum
-	if cfg.Checksum {
+	// pipeline checksum only when enabled and is not incremental snapshot repair mode cuz incremental doesn't have
+	// enough information in backup meta to validate checksum
```

Comment:
we don't use the collection of `file-level` checksums to match the `table-level` checksum in the incremental backup. However, we use the `table-level` checksum in the incremental backup to match the `table-level` checksum in the incremental restore.

Reply:
could you point me to the code, I thought table level checksum for incremental is not calculated during backup 
```
skipChecksum := !cfg.Checksum || isIncrementalBackup
```

Reply:
Get it.